sdk/java-sdk-testkit/src/main/java/kalix/javasdk/testkit/KalixTestKit.java

```diff
@@ -71,7 +71,7 @@ public static class Settings {
      */
     @Deprecated
     public Settings(final Duration stopTimeout) {
-      this(stopTimeout, "self", true);
+      this(stopTimeout, "self", false);
```

Comment:
To align with the docker-compose settings, I think this should also be disabled by default. 

All our sample tests were passing because we configured the default ACL to be `principal = INTERNET`. 

Users starting a new service and trying to write an integration tests will get blocked. Worst, there is no clear message in the logs explaining that the tests are failing because of the ACL restriction. It just doesn't work and users are left without any clue.